### DISCUSSION TITLE
Clear cache

### DISCUSSION THREAD

**Discussion by artemklishch** [vitest-dev/vitest#4028] [Q&A]
Hello!
Is there any clear cache command for 'vitest' test?
Thank you!

**Comment by stephane303** [comment]
Or at least document where all the cache could be

**Reply by sheremet-va** [reply]
Should be mentioned in the next docs update via https://github.com/vitest-dev/vitest/commit/a663c7b5c83a11bd8f3bf4db78e58ed9dbd94be4

TLDR: cache is stored in [`cacheDir`](https://vitejs.dev/config/shared-options.html#cachedir) option (`node_modules/.vite` by default)
